reactive.js

```diff
@@ -0,0 +1,56 @@
+/* reactive engine. the code here combines functions exposed by the
+ * library into a single function to be run. i keep all code depending
+ * on Bacon here, so if somebody will want to stop using Bacon in the
+ * future, it will be easier */
+var lib = require('./lib2');
+var Bacon = require('bacon');
+
+// this is an adapter function. it gets a Pouch changes object and it
+// returns a Bacon stream
+function changesToStream(changes) {
+  return Bacon.fromBinder(function(sink) {
+    changes
+      .on('change', sink)
+      .on('error', function(text) {
+        sink(new Bacon.Error(text));
+      })
+      .on('complete', function() {
+        sink(new Backn.End());
```

Comment:
the code in this file is ready, except for bugs i can find. i tried to write it as clearly as possible and to add comments, so please ask about specific aspects which are not clear. i think that the biggest problem here is the abstraction implied by using Bacon, it took some time also for me to get acquainted with it, but the result is very concise as you can see, so i hope that this is a good way to tackle this problem. probably the most confusing aspect in Bacon is the difference between streams and properties, but it would not make sense to copy the documentation here